src/licensedcode/data/licenses/g10-permissive.LICENSE

```diff
@@ -0,0 +1,18 @@
+---
+key: g10-permissive
+short_name: g10-permissive
+name: g10-permissive
+category: Permissive
+owner: g10 Code GmbH
+homepage_url: https://changelogs.ubuntu.com/changelogs/pool/main/d/dbus/dbus_1.10.6-1ubuntu3.6/copyright
+spdx_license_key: LicenseRef-scancode-g10-permissive
+standard_notice: 'License: g10-permissive'
```

Comment:
That's not a standard notice IMHO... and if you want this detected it should be a rule instead